packages/smooth_app/lib/helpers/picture_capture_helper.dart

```diff
@@ -13,32 +19,52 @@ Future<bool> uploadCapturedPicture(
   required Uri imageUri,
 }) async {
   final AppLocalizations appLocalizations = AppLocalizations.of(context);
-  final SendImage image = SendImage(
-    lang: ProductQuery.getLanguage(),
+  final LocalDatabase localDatabase = context.read<LocalDatabase>();
+  final String uniqueId = _getUniqueId(imageField, barcode);
+  final BackgroundImageInputData backgroundImageInputData =
+      BackgroundImageInputData(
+    processName: IMAGE_UPLOAD_TASK,
+    uniqueId: uniqueId,
     barcode: barcode,
-    imageField: imageField,
-    imageUri: imageUri,
+    imageField: imageField.value,
+    imageUri: File(imageUri.path).path,
```

Comment:
Mmm that's weird to have a path… for an Uri.
I think something is wrong in terms of naming

Reply:
Maybe, you should change imageUri to imagePath